package.json

```diff
@@ -36,7 +36,7 @@
     "detox": "^12.11.1",
     "dotenv": "^7.0.0",
     "react": "16.8.3",
-    "react-native": "0.59.2",
+    "react-native": "0.59.9",
```

Comment:
Would you also like to update the other dependencies, like in https://react-native-community.github.io/upgrade-helper/?from=0.59.2&to=0.59.9?

Reply:
They are already up to date. You can run `yarn upgrade-interactive --latest` and see that none of these dependencies show up. 

You can also verify by checking the yarn.lock file.